modules/logging-loki-logcli.adoc

```diff
@@ -0,0 +1,32 @@
+// Module is included in the following assemblies:
+// logging/logging-loki-ocp.adoc
```

Comment:
I don't see it included in that file. Is this left-over from a previous version?